pkg/policy/l4.go

```diff
@@ -177,6 +177,34 @@ func (l4 *L4Filter) ToKeys(direction trafficdirection.TrafficDirection, identity
 	return keysToAdd
 }
 
+// ProcessDatapathEntryFunc is a function type which takes some action based
+// upon each datapath key+value that is generated from the L4Filter.
+type ProcessDatapathEntryFunc func(Key, MapStateEntry)
+
+func (l4 *L4Filter) ForEachDatapathEntry(
```

Comment:
Documentation needed

Reply:
This was fixed in a future commit, resolving.

Reply:
It's in a later commit.